tensorflow/core/kernels/data/concatenate_dataset_op.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef TENSORFLOW_CORE_KERNELS_DATA_CONCATENATE_DATASET_OP_H_
+#define TENSORFLOW_CORE_KERNELS_DATA_CONCATENATE_DATASET_OP_H_
+
+#include "tensorflow/core/framework/dataset.h"
+
+namespace tensorflow {
+namespace data {
+
+class ConcatenateDatasetOp : public BinaryDatasetOpKernel {
+ public:
+  static constexpr const char kDatasetType[] = "Concatenate";
```

Comment:
@jsimsa Find another style to declare the static constant name as [here](https://github.com/tensorflow/tensorflow/blob/a26d43a5f8e06f5236d4117bbda702fac26ae93a/tensorflow/core/framework/function.h#L318). Using that style, `kDatasetType` can be declared as `static constexpr const char* const kDatasetType = "Concatenate";` and the re-declaration in the .cc file will not be needed. 

Which style do you think it is better?

Reply:
I like the one that does not require .cc re-declaration.

Reply:
Me too. Will change them in the next PR.